app/models/application.rb

```diff
@@ -101,10 +113,10 @@ def commit_history
         {
           deployed_to:,
           tags:,
-          message: commit[:commit][:message].split(/\n/)[0],
-          author: commit.dig(:commit, :author, :name),
-          sha: commit[:sha],
-          github_url: "#{repo_url}/commit/#{commit[:sha]}",
+          message: commit.message.split(/\n/)[0],
```

Comment:
Does this need to be a regex?
```suggestion
          message: commit.message.split("\n")[0],
```

Reply:
or `.lines[0]` maybe?